src/BSONIter.jl

```diff
@@ -97,35 +98,29 @@ end
 
 # Iterator
 
-start(bsonObject::BSONObject) = begin
+Base.start(bsonObject::BSONObject) = begin
```

Comment:
I'd get rid of the begin return end, i.e.:
`Base.start(obj::BSONObject) = BSONIter(obj)`